src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -105,7 +106,7 @@ public Task<string> RunDarcAsync(params string[] args)
         {
             return TestHelpers.RunExecutableAsync(_parameters.DarcExePath, args.Concat(new[]
             {
-                "-p", _parameters.MaestroToken,
+                "--p", _parameters.MaestroToken,
```

Comment:
I'm kind of surprised this syntax works (I checked). Oftenpassing -- to prefix a shortened version of a parameter is incorrect (e.g. git)